### ISSUE TITLE
Wrong return type for `hasOne()`

### ISSUE THREAD

**Issue by edwinvdpol** [Soyhuce/next-ide-helper#79]
Hi there,

I noticed a wrong return type when using the `hasOne()` relationship.

The relationship below...

```php
public function entity(): HasOne
{
    return $this->hasOne(Entity::class);

    // Or like ...
    return $this->hasOne(Entity::class)->latestOfMany();
}
```

Will generate the following docblock:
```php
/**
 * @property-read \App\Models\Entity $entity
 */
```

The thing is, this _can_ return `null`:
```php
// This shows null when the relationship if not present.
dd($class->entity);
```

Is there any change this can be changed/fixed?
I tried to fix it, but I'm not _that_ good with the `ReflectionClass`. :sweat_smile:

I am using Laravel 10, and the relationship is defined in a `Trait`.

Best regards,
Edwin

**Comment by bastien-phi** [comment]
Hi @edwinvdpol ,

You're right ! I just pushed the fix. I will publish a new version very soon.

In the meantime, you can also override your attributes or relation types using this package configuration. 

```php
return [
    /**
     * Configure models command.
     */
    'models' => [
        /**
         * Add override to define manually attribute type for some models.
         *
         * Overrides should be declared as follow :
         * MyModel::class => [
         *      'attribute' => 'type'
         * ]
         *
         * Prepend the type with '?' to mark it as nullable.
         */
        'overrides' => [
            YourModel::class => [
                'entity' => '?'.Entity::class,
            ]
        ],
    ]        
];
```

**Comment by bastien-phi** [comment]
Fixed in 0.13.0

**Comment by edwinvdpol** [comment]
Thank you, @bastien-phi for the quick fix!
